### PR TITLE
dd AceLuodan's file

### PR DESCRIPTION
**Pull request by AceLuodan**
created a branch, created a file and made a commit, and opened a pull request

### CODE REVIEW

**Review by AceLuodan — COMMENTED** [review]
leave something

**Review by AceLuodan — COMMENTED** [review]
review

**Review by AceLuodan — COMMENTED** [review]
cxv

File: _posts/0000-01-02-AceLuodan.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```
Comment (AceLuodan) [inline _posts/0000-01-02-AceLuodan.md:5]: for new

Comment (AceLuodan) [inline _posts/0000-01-02-AceLuodan.md:5]: ad

**Review by AceLuodan — COMMENTED** [review]
sad

File: _posts/0000-01-02-AceLuodan.md (hunk 1 above)
Comment (AceLuodan) [inline _posts/0000-01-02-AceLuodan.md:5]: cxv

Comment (AceLuodan) [inline _posts/0000-01-02-AceLuodan.md:5]: g

**Comment by brianamarie** [comment]
Hi @AceLuodan! It looks like Learning Lab is expecting more work in this pull request. Are you able to reopen this pull request? If so, then the next steps would be to click the "files changed" tab at the top, and edit the file on line 5.

Let me know if you want or need any help! 😄

**Pull request closed without merging**